apps/dokploy/templates/teable/docker-compose.yml

```diff
@@ -0,0 +1,83 @@
+version: '3.9'
+
+services:
+  teable:
+    image: ghcr.io/teableio/teable:1.3.1-alpha-build.460
+    restart: always
+    ports:
+      - ${TEABLE_PORT}
+    volumes:
+      - teable-data:/app/.assets:rw
+      # you may use a bind-mounted host directory instead,
+      # so that it is harder to accidentally remove the volume and lose all your data!
+      # - ./docker/teable/data:/app/.assets:rw
+    environment:
+      - TZ=${TIMEZONE}
+      - NEXT_ENV_IMAGES_ALL_REMOTE=true
+      - PUBLIC_ORIGIN=${PUBLIC_ORIGIN}
+      - PRISMA_DATABASE_URL=${PRISMA_DATABASE_URL}
+      - PUBLIC_DATABASE_PROXY=${PUBLIC_DATABASE_PROXY}
+      - BACKEND_MAIL_HOST=${BACKEND_MAIL_HOST}
+      - BACKEND_MAIL_PORT=${BACKEND_MAIL_PORT}
+      - BACKEND_MAIL_SECURE=${BACKEND_MAIL_SECURE}
+      - BACKEND_MAIL_SENDER=${BACKEND_MAIL_SENDER}
+      - BACKEND_MAIL_SENDER_NAME=${BACKEND_MAIL_SENDER_NAME}
+      - BACKEND_MAIL_AUTH_USER=${BACKEND_MAIL_AUTH_USER}
+      - BACKEND_MAIL_AUTH_PASS=${BACKEND_MAIL_AUTH_PASS}
+    networks:
+      - dokploy-network
+      - teable-standalone
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.${HASH}.rule=Host(`${TEABLE_HOST}`)"
+      - "traefik.http.services.${HASH}.loadbalancer.server.port=${TEABLE_PORT}"
+    depends_on:
+      teable-db-migrate:
+        condition: service_completed_successfully
+
+  teable-db:
+    image: postgres:15.4
+    restart: always
+    ports:
+      - '42345:5432'
+    volumes:
+      - teable-db:/var/lib/postgresql/data:rw
```

Comment:
Remove the `:rw`, if you deploy and try to deploy again it will become unnaccesible and it will say bad gateway but deleting that will remove the issue